roles/network_plugin/flannel/templates/cni-flannel.yml.j2

```diff
@@ -34,7 +34,8 @@ data:
       "Backend": {
         "Type": "{{ flannel_backend_type }}"{% if flannel_backend_type == "vxlan" %},
         "VNI": {{ flannel_vxlan_vni }},
-        "Port": {{ flannel_vxlan_port }}
+        "Port": {{ flannel_vxlan_port }},
+        "DirectRouting": {{ flannel_vxlan_direct_routing | to_json }}
```

Comment:
I could not find some document to verify this option is valid.
Could you provide it if possible?

I found https://github.com/flannel-io/flannel/issues/1554 and they said it is not valid.

Reply:
It's in the the flannel backend documentation for VXLAN: https://github.com/flannel-io/flannel/blob/master/Documentation/backends.md#vxlan

It's been there for quite some time AFAIK because I had it enabled on an old k8s 1.18 cluster.

Reply:
Thanks for your explanation, that is perfect answer for me.

/ok-to-test
/approve